infra/base-images/base-runner/coverage

```diff
@@ -55,6 +55,9 @@ PATH_EQUIVALENCE_ARGS="-path-equivalence=/,$OUT"
 LLVM_COV_COMMON_ARGS="$PATH_EQUIVALENCE_ARGS \
     -ignore-filename-regex=.*src/libfuzzer/.* $COVERAGE_EXTRA_ARGS"
 
+# Options to extract branch coverage.
+BRANCH_COV_ARGS="--show-branches=count --show-expansions"
```

Comment:
Does this significantly increase size of the resulting reports?

Reply:
It definitely increases the report size as it is exposing coverage information for each branch in the code. The amount of increase is project dependent, for the sample project (`htslib`) that I checked, the increase is 2X.